hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -43,14 +43,22 @@ message DatanodeDetailsProto {
     // network name, can be Ip address or host name, depends
     optional string networkName = 6;
     optional string networkLocation = 7; // Network topology location
-    optional string version = 8;         // Datanode version
```

Comment:
@runitao Can we mark these fields as 'deprecated', and not set them in the code, rather than removing them. This will likely backward compatibility when we have a case to add a new field in the future.

Reply:
@avijayanhwx  Thanks for review. No problem. But I need to confirm. 
Should I only keep version and setupTime(these two fields added in 0.6.0) or keep these four fields?

Reply:
@runitao When were these fields added? Were they part of 0.6.0 release?

Reply:
I got it, I will keep version and setupTime fields.

Reply:
Fix that, PTAL @avijayanhwx, thanks.

Reply:
@runitao HDDS-3798 is marked for 0.7.0, and hence I will revert it in branch **ozone-0.6.0**.

Reply:
@avijayanhwx Great. The code will be more clean, I have removed the four fields from DatanodeDetailsProto.